README.adoc

```diff
@@ -1029,6 +1029,20 @@ User.all.size
 User.all.length
 ----
 
+=== `where.not` with multiple attributes
+
+Until Rails 6.1, when negating condition with multiple attributes using `where.not`, it works like NOR (NOT(A) AND NOT(B)). This does not follow boolean algebra law `NOT (A AND B) = (NOT(A) OR NOT(B))` and could lead to confusing results.
```

Comment:
Is the example with `NOR` correct here? It operates on two values.

Reply:
I meant `NOT(A) AND NOT(B)` is `NOR`. Anyways I think the sentence is better off without mentioning `NOR` and `NAND` to avoid confusion. Removed it.